winch/codegen/src/isa/aarch64/masm.rs

```diff
@@ -385,9 +385,11 @@ impl Masm for MacroAssembler {
         size: OperandSize,
         trap: TrapCode,
     ) -> Result<()> {
-        self.add(dst, lhs, rhs, size)?;
-        self.asm.trapif(Cond::Hs, trap);
-        Ok(())
+        self.with_aligned_sp(|masm| {
```

Comment:
Can we add a comment here noting why the SP alignment is needed? (Is it required to be aligned at all potentially trapping ops?)

Reply:
Done, thanks!